### ISSUE TITLE
[Y1 Summer 2026] **The Strict Boundary:** Immediately split the BRCA dataset into an **80% Training/Validation Set** ...

### ISSUE THREAD

**Issue by AdamCankaya** [AdamCankaya/PhDNeural#185]
## Context

- **Year:** 1 — BRCA Anchor & Two-Stage Fusion (2026–2027)
- **Semester:** Summer 2026 — TCGA Sourcing & Data Foundation
- **Phase:** 1 — The Anchor (BRCA PoC)
- **Step:** 1 — BRCA Multi-Omic Sourcing & Split
- **Goal:** Acquire the most feature-dense dataset to stress-test every branch of the neural network.

## Task

**The Strict Boundary:** Immediately split the BRCA dataset into an **80% Training/Validation Set** and a locked **20% Holdout Test Set** before any preprocessing.

---
_Auto-synced from phd_master_plan.md_
